### PR TITLE
Added german language pack

### PR DESCRIPTION
**Pull request by gamersi**
I made german(deutsch) translations for muffon

### CODE REVIEW

File: shared/locales/de.json [hunk 1]
```diff
@@ -0,0 +1,672 @@
...
+        "search": "Von der Suche",
+        "folder": "Von dem Ordner",
+        "account": "Von dem Account",
+        "library": "Von der Bibliothek"
+      },
+      "searchIn": {
+        "sources": "In Quellen suchen"
+      },
+      "searchFor": {
+        "video": "nach Videos suchen",
```
Comment (staniel359) [inline shared/locales/de.json]: @gamersi Sorry, I don't know German.
Shouldn't it start with capital letter?

File: shared/locales/de.json [hunk 2]
```diff
@@ -0,0 +1,672 @@
...
+    },
+    "feed": {
+      "global": "Globaler feed"
+    },
+    "about": {
+      "license": "{license} Lizenz",
+      "homepage": "Startseite",
+      "contact": "Kontakt"
+    },
+    "listeners": "{count} zuhörer | {count} zuhörer",
```
Comment (staniel359) [inline shared/locales/de.json]: @gamersi Are single and plural forms the same?
I so, you can leave just one variant.

File: shared/locales/de.json [hunk 3]
```diff
@@ -0,0 +1,672 @@
...
+      "joinedDesc": "Am neuesten",
+      "joinedAsc": "Am ältesten",
+      "membersCountDesc": "Meiste Mitglieder",
+      "membersCountAsc": "Wengiste Mitglieder"
+    },
+    "error": "Fehler",
+    "more": "Mehr",
+    "loading": "Lädt..."
+  }
+
```
Comment (staniel359) [inline shared/locales/de.json]: @gamersi Seems like there's a trailing whitespace here.

**Review by staniel359 — CHANGES_REQUESTED** [review]

File: shared/locales/de.json (hunk 2 above)
Comment (gamersi) [reply shared/locales/de.json]: there are just like in english some plural words that are the same, in this case it is one

ein zuhörer (one listener)
drei zuhörer (three listeners)

File: shared/locales/de.json (hunk 1 above)
Comment (gamersi) [reply shared/locales/de.json]: in this case you are actually right

File: shared/locales/de.json (hunk 3 above)
Comment (gamersi) [reply shared/locales/de.json]: if you mean the one in line 672 i removed it now

File: shared/locales/de.json (hunk 1 above)
Comment (staniel359) [reply shared/locales/de.json]: @gamersi Actually I meant `nach Videos suchen`.
Is this correct in German to start sentences with lowercase letter?

File: shared/locales/de.json (hunk 2 above)
Comment (staniel359) [reply shared/locales/de.json]: @gamersi I see, thank you.
Then please leave just `"listeners": "{count} zuhörer"`

File: shared/locales/de.json (hunk 1 above)
Comment (gamersi) [reply shared/locales/de.json]: Yeah I am sorry I misunderstood. Every Sentence starts uppercase in german, so both are uppercase

image

File: shared/locales/de.json (hunk 2 above)
Comment (gamersi) [reply shared/locales/de.json]: Ok will do that

Comment (gamersi) [reply shared/locales/de.json]: changed

File: shared/locales/de.json (hunk 1 above)
Comment (gamersi) [reply shared/locales/de.json]: changed

Comment (staniel359) [reply shared/locales/de.json]: @gamersi Then could you also fix other such translations?
E.g., `in Bibliothek anzeigen`, `von Favoriten entfernen`...

File: shared/locales/de.json (hunk 2 above)
Comment (gamersi) [reply shared/locales/de.json]: > @gamersi I see, thank you. Then please leave just `"listeners": "{count} zuhörer"`

should I also remove it in line 323 and other lines where there are pluralwords?

File: shared/locales/de.json (hunk 1 above)
Comment (gamersi) [reply shared/locales/de.json]: Did that and found a few other small spelling & typing mistakes

File: shared/locales/de.json (hunk 2 above)
Comment (staniel359) [reply shared/locales/de.json]: @gamersi Yes, please.

File: shared/locales/de.json (hunk 1 above)
Comment (staniel359) [reply shared/locales/de.json]: @gamersi Nice, thank you.

File: shared/locales/de.json (hunk 3 above)
Comment (staniel359) [reply shared/locales/de.json]: @gamersi Thanks.

File: shared/locales/de.json (hunk 2 above)
Comment (gamersi) [reply shared/locales/de.json]: > @gamersi Yes, please.

Should be all fixed now

Comment (staniel359) [reply shared/locales/de.json]: @gamersi Thank you.

**Review by staniel359 — APPROVED** [review]
@gamersi Great job, thank you very much!

**Comment by staniel359** [comment]
@gamersi If there're any changes, can I mention you for update?

**Comment by gamersi** [comment]
Yes, you can

**Comment by staniel359** [comment]
@gamersi Good, thanks.

**Comment by staniel359** [comment]
@gamersi Hello.
Could you also add translations to [website](https://github.com/staniel359/muffon-website/tree/main/src/data)?

**Comment by gamersi** [comment]
> @gamersi Hello.
> 
> Could you also add translations to [website](https://github.com/staniel359/muffon-website/tree/main/src/data)?

Will do that when I am home later today

**Comment by staniel359** [comment]
@gamersi Thank you.

**Pull request merged**